data/modules/odgi/v0.4.1/README.md

```diff
@@ -0,0 +1,39 @@
+---
+Name: odgi
+URL: https://github.com/vgteam/odgi
```

Comment:
This readme file isn't parsed anywhere so this YAML front matter isn't used. However, it renders fine on GitHub so doesn't hurt to have it 👍🏻